app/src/controllers/userController.ts

```diff
@@ -0,0 +1,127 @@
+import mongoose from 'mongoose';
+import { UserSchema } from '../models/userModel';
+import { Request, Response } from 'express';
+
+const User = mongoose.model('User', UserSchema);
+
+/**
+ * CRUD-Controller for user models
+ */
+export class UserController {
+
+    /**
+     * lists all available users
+     * @param  {Request} req - express request object
+     * @param  {Response} res — express response object
+     * @return json object with statusCode and list of users
+     */
+    public async index(req: Request, res: Response) {
+        try {
+            const users = await User.find();
+            res.json({
```

Comment:
```suggestion

            res.json({
```

Reply:
you need to add `eslint` package with [typescript-eslint](https://github.com/typescript-eslint/typescript-eslint) plugins. And with [our config](http://github.com/codex-team/eslint-config)

Example: https://github.com/codex-team/codex.music/blob/master/.eslintrc.js

And then fix all `eslint` issues.

Reply:
I installed the linter and the configuration. 
The only warnings persisting are those:

```
/app/src/controllers/userController.ts
  3:10  warning  'Request' is defined but never used   no-unused-vars
  3:19  warning  'Response' is defined but never used  no-unused-vars
```

Is there a way to fix those because the definitions are needed for the typescript definitions?